guidelines.md

```diff
@@ -0,0 +1,6 @@
+# Some Tips for Writing Quick Tips
+- Explain everything as if your audience has no background on anything
+- Short and sweet
+- Bullets are your best friend
+- Break things down into sections
```

Comment:
mention linking other resources. I'd rebring up the links you mentioned at the beginning in the end as well tbh